packages/checkbox/test/visual/lumo/checkbox.test.js

```diff
@@ -36,25 +32,47 @@ describe('checkbox', () => {
     await visualDiff(div, 'indeterminate');
   });
 
-  it('disabled checked', async () => {
-    element.disabled = true;
-    element.checked = true;
-    await visualDiff(div, 'disabled-checked');
+  it('focus-ring', async () => {
+    await sendKeys({ press: 'Tab' });
+    await visualDiff(div, 'focus-ring');
   });
 
-  it('disabled indeterminate', async () => {
-    element.disabled = true;
-    element.indeterminate = true;
-    await visualDiff(div, 'disabled-indeterminate');
-  });
+  describe('disabled', () => {
+    beforeEach(() => {
+      element.disabled = true;
+    });
 
-  it('empty', async () => {
-    element.textContent = '';
-    await visualDiff(div, 'empty');
+    it('basic', async () => {
+      await visualDiff(div, 'disabled');
+    });
+
+    it('checked', async () => {
+      element.checked = true;
+      await visualDiff(div, 'disabled-checked');
+    });
+
+    it('indeterminate', async () => {
+      element.indeterminate = true;
+      await visualDiff(div, 'disabled-indeterminate');
+    });
   });
 
-  it('RTL', async () => {
-    document.documentElement.setAttribute('dir', 'rtl');
-    await visualDiff(div, 'rtl');
+  describe('RTL', () => {
+    before(() => {
+      document.documentElement.setAttribute('dir', 'rtl');
+    });
+
+    after(() => {
+      document.documentElement.removeAttribute('dir');
+    });
+
+    it('basic', async () => {
+      await visualDiff(div, 'rtl');
+    });
+
+    it('empty', async () => {
+      element.label = '';
```

Comment:
**note:** The margin between the label and the input is styled separately for RTL and LTL modes. The margin should disappear when there is no label. That being so, I thought it would be reasonable to test the empty label case for both modes.